dumper/app/gradle.lockfile

```diff
@@ -170,6 +170,8 @@ junit:junit:4.13.2=testCompileClasspath,testRuntimeClasspath
 net.bytebuddy:byte-buddy-agent:1.12.19=testCompileClasspath,testRuntimeClasspath
 net.bytebuddy:byte-buddy:1.12.19=testCompileClasspath,testRuntimeClasspath
 net.java.dev.jna:jna:5.17.0=runtimeClasspath,testFixturesRuntimeClasspath,testRuntimeClasspath
+net.java.dev.jna:jna-platform:5.16.0=compileClasspath,runtimeClasspath,testFixturesRuntimeClasspath,testRuntimeClasspath,testCompileClasspath
```

Comment:
You excluded this deps but add it to the lock file. It's unusual. Please double check